master/internal/api_user.go

```diff
@@ -170,12 +173,19 @@ func (a *apiServer) GetUsers(
 		nameFilterExpr := "%" + req.Name + "%"
 		query.Where("u.display_name ILIKE ? OR u.username ILIKE ?", nameFilterExpr, nameFilterExpr)
 	}
-	if req.Admin != nil {
+	if req.Admin != nil && !config.GetAuthZConfig().IsRBACEnabled() {
 		query.Where("u.admin = ?", *req.Admin)
 	}
 	if req.Active != nil {
 		query.Where("u.active = ?", *req.Active)
 	}
+	if len(req.RoleIdAssignedDirectlyToUser) != 0 && config.GetAuthZConfig().IsRBACEnabled() {
```

Comment:
Sure.
I was thinking all other random params were ignored and it makes sense to treat `role_id_assigned_directly_to_user` the same, since it's basically an invalid argument in OSS